accepted/0000-yomi-initial-integration.md

```diff
@@ -0,0 +1,427 @@
+- Feature Name: Initial integration with Yomi (MVP)
+- Start Date: 2019-10-08
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Yomi (Yet one more installer) is an installer for openSUSE (Leap,
+Tumbleweed, MicroOS) and SLE (WIP) expressed as a set of Salt
+states. This RFC propose the integration of Yomi for provisioning of
+new systems in Uyuni.
+
+# Motivation
+[motivation]: #motivation
+
+Currently in Uyuni / SUMA we have several ways for installing a new
+system:
+
+- For RedHat based systems we can use the Web UI to create, update and
+  manage Kickstart files. Those files can be designed to be applied
+  for different nodes. (IS THIS TRUE?)
+
+- For [open]SUSE based systems we can upload an AutoYaST file designed
+  for a single type on system.
+
+- We can leverage Cobbler manage the PXE-Boot server and automate the
+  installation of new systems (via Kickstart, AutoYaST and Cheetah or
+  Jinja2 template engine)
+
+- SUMA Manager for Retail allows the installation of new systems
+  dumping a partition image (generated by KIWI from a given template)
+  into the local root partition. For this case the PXE boot stack will
+  be different that the one provided by Cobbler
+
+For one side we can see some unbalance between the Kickstart and the
+AutoYaST integration: the Kickstart template edition is more naturally
+integrated into the UI.
+
+For other side we are moving Uyuni forward into a better and more
+consistent integration with SaltStack, but only the retail version
+(based on images) is using this configuration management system.
+
+To improve this situation we propose the integration of the Yomi
+installer into Uyuni. Yomi re-formulate the installation process as a
+set of Salt states. In this model the pillar will contain the
+definition of:
+
+- Partitions and file systems used in the node
+
+- Bootloader parameters
+
+- Repositories and packages installed
+
+- Services enabled and disabled
+
+- Users and certificates
+
+As we can see, this proposal is (1) moving closer the features of
+Kickstart based systems into the SUSE family, and (2) using SaltStack
+to orchestrate the new installation.
+
+# Detailed design
+[design]: #detailed-design
+
+## Integrating Yomi as a Salt state
+
+The core of the proposal is to add Yomi into the catalog of formulas
+available in Uyuni.
+
+As Yomi is indeed a Salt formula, this integration can be done
+directly, installing the `yomi-formula` package into the Manager.
+
+The user can later decide to add the Yomi state into the
+auto-generated top file for this system, fill the pillar data using
```

Comment:
The wording seems strange to me. What does it means to add Yomi to the auto-generated top file? I know what you mean, but why not write 'The user can later decide to enable Yomi formula for registered system, ...'

Reply:
Yes, your proposal is more clear. Replaced!